web/src/routes/(user)/search/[[photos=photos]]/[[assetId=id]]/+page.svelte

```diff
@@ -149,10 +148,7 @@
 
   // eslint-disable-next-line svelte/valid-prop-names-in-kit-pages
   export const loadNextPage = async (force?: boolean) => {
-    if (!nextPage || searchResultAssets.length >= MAX_ASSET_COUNT) {
-      return;
-    }
-    if (isLoading && !force) {
+    if (!nextPage || isLoading && !force) {
```

Comment:
should be as intended

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_AND#operator_precedence